compiler/optimizer/ValuePropagationCommon.cpp

```diff
@@ -1159,6 +1158,7 @@ void OMR::ValuePropagation::transformArrayCopyCall(TR::Node *node)
         srcOffHigh >= 0 && dstOffHigh >= 0 && copyLenHigh >= 0)
       {
       transformTheCall = true;
```

Comment:
@jdmpapin I agree. We should move towards a more conservative approach. I've made the change in 5273acf.